src/sentry/static/sentry/app/components/sidebar/index.jsx

```diff
@@ -412,6 +439,11 @@ class Sidebar extends React.Component {
         {hasOrganization && (
           <SidebarSectionGroup>
             <SidebarSection>
+              {HookStore.get('sidebar:bottom-items').length &&
```

Comment:
You can just use the `Hook` component here.

Reply:
oo yeah so I started out using `Hook` but once the component mounts updates to props don't propagate for some reason I need to look into which affects the collapsed state :/

Reply:
/cc @billyvg is this a known issue with the `Hook`?